source/actors/pacorn.cpp

```diff
@@ -58,23 +58,36 @@ void PAcorn::vf18C() {
 }
 
 void PAcorn::playBlockReleaseUpAnim() {
-    this->model->playSklAnim("out", 1);
+    this->model->playSklAnim("out", 0);
+    this->model->sklAnims[0]->frameCtrl.shouldLoop(false);
+    this->_1810 = 1;
```

Comment:
What is this->_1810?

Reply:
I am not exactly sure, but I found it in the hammer flower code and it works so I am not complaining. Probably animation id thingy, but idk.